src/Ryujinx.Graphics.Nvdec.Vp9/LoopFilter.cs

```diff
@@ -134,29 +129,28 @@ internal static class LoopFilter
         };
 
         // These are used for masking the left and above borders.
+#pragma warning disable IDE0051 // Remove unused private member
         private const ulong LeftBorder = 0x1111111111111111UL;
         private const ulong AboveBorder = 0x000000ff000000ffUL;
+#pragma warning restore IDE0051
 
         // 16 bit masks for uv transform sizes.
-        private static readonly ushort[] Left64X64TxformMaskUv = new ushort[]
-        {
+        private static readonly ushort[] _left64X64TxformMaskUv = {
             0xffff,  // TX_4X4
             0xffff,  // TX_8x8
             0x5555,  // TX_16x16
             0x1111,  // TX_32x32
         };
 
-        private static readonly ushort[] Above64X64TxformMaskUv = new ushort[]
-        {
+        private static readonly ushort[] _above64X64TxformMaskUv = {
             0xffff,  // TX_4X4
             0xffff,  // TX_8x8
             0x0f0f,  // TX_16x16
             0x000f,  // TX_32x32
         };
 
         // 16 bit left mask to shift and set for each uv prediction size.
-        private static readonly ushort[] LeftPredictionMaskUv = new ushort[]
-        {
+        private static readonly ushort[] _leftPredictionMaskUv = {
             0x0001,  // BLOCK_4X4,
```

Comment:
```suggestion
            0x0001, // BLOCK_4X4,
```
Same for other comments (in the whole file)